docs/pages/documentation/data/Table.vue

```diff
@@ -42,15 +44,18 @@
                         :bordered="isBordered"
                         :striped="isStriped"
                         :narrowed="isNarrowed"
+                        :hasDetails="hasDetails"
                         :checkable="isCheckable"
                         :loading="isLoading"
                         :mobile-cards="hasMobileCards"
                         :paginated="isPaginated"
                         :per-page="perPage"
                         :pagination-simple="isPaginationSimple"
+                        default-sort="user.first_name"
                         :selected.sync="selected"
-                        :checked-rows.sync="checkedRows"
-                        default-sort="user.first_name">
+                        @details-opened="onOpenedDetail"
```

Comment:
Better rename to 'details-open' and 'details-close' to keep like all others events.